include/xrpl/protocol/Protocol.h

```diff
@@ -95,6 +95,12 @@ std::size_t constexpr maxDIDAttestationLength = 256;
 /** The maximum length of a domain */
 std::size_t constexpr maxDomainLength = 256;
 
+/** The maximum length of MPTokenMetadata */
+std::size_t constexpr maxMPTokenMetadataLength = 1024;
+
+/** The maximum amount of MPTokenIssuance */
```

Comment:
Depending on the outcome of our discussion around issuance max values, we might want to adjust this comment to something like this:

```suggestion
/** The maximum amount of an MPT inside of an STAmount */
```

Reply:
Ugg.. my _actual_ comment didn't make it. Let me re-post.

Reply:
@shawnxie999 This comment can be resolved.